src/AmbientSounds.Uwp/App.xaml

```diff
@@ -13,16 +13,8 @@
             </ResourceDictionary.MergedDictionaries>
 
             <ResourceDictionary.ThemeDictionaries>
-                <ResourceDictionary x:Key="Light">
-                    <Color x:Key="SystemAccentColor">#e37700</Color>
-                    <Color x:Key="SystemAccentColorLight1">#FFE8881E</Color>
-                    <Color x:Key="SystemAccentColorLight2">#FFED993C</Color>
-                    <Color x:Key="SystemAccentColorLight3">#FFF1AA5A</Color>
-                    <Color x:Key="SystemAccentColorDark1">#FFCE6500</Color>
-                    <Color x:Key="SystemAccentColorDark2">#FFB85300</Color>
-                    <Color x:Key="SystemAccentColorDark3">#FFA34100</Color>
-
-                    <LinearGradientBrush x:Key="BlueGradientBrush" StartPoint="0,0.5" EndPoint="1,0.5">
+				<ResourceDictionary x:Key="Light">
+					<LinearGradientBrush x:Key="BlueGradientBrush" StartPoint="0,0.5" EndPoint="1,0.5">
```

Comment:
can you fix the tabbing here and in the other set of lines you changed?

Reply:
Seems to be a visual studio issue, will fix tomorrow